Automattic/jetpack
# Charts: Adding pie charts

Fixes https://github.com/Automattic/jetpack-roadmap/issues/2047

## Proposed changes:

* adding new chart type - pie and daughnut
* extending tooltips with a hook to abstract tooltip event handlers
* adding new sample data to storybook stories

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* install Jetpack packages if needed (this PR adda one new packages for future responsive UI tasks) `pnpm i`
* navigate to the directory and run storybook: `cd projects/js-packages/charts` and `pnpm run storybook`
* verify that existing charts are working and that the new pie chart is available
* verify that the pie chart works as expected and applied colours to different sections

![SCR-20241212-lgic](https://github.com/user-attachments/assets/52dacd14-fcae-49e1-9682-5b4112990996)
